website/addons/owncloud/static/user-cfg.js

```diff
@@ -0,0 +1,9 @@
+var $osf = require('js/osfHelpers');
+var OwnCloudViewModel = require('./owncloudUserConfig.js').OwnCloudViewModel;
+
+var url = '/api/v1/settings/owncloud/';
+
+var owncloudViewModel = new OwnCloudViewModel(url);
+$osf.applyBindings(owncloudViewModel, '#owncloudAddonScope');
```

Comment:
This viewmodel initialization is what's being done by `OwnCloudUserConfig`. Why not `require` and instantiate that instead?